logstash_formatter/__init__.py

```diff
@@ -89,9 +89,15 @@ def format(self, record):
                 fields['exception'] = formatted
             fields.pop('exc_info')
 
-        if 'exc_text' in fields and not fields['exc_text']:
+        if 'exc_text' in fields:
             fields.pop('exc_text')
 
+        if 'message' in fields:
+            fields.pop('message')
+        if 'levelname' in fields:
+            fields['loglevel'] = fields.get('levelname', '')
```

Comment:
levelname comes in from the python logger. we want it to be loglevel we are popping levelname